cdap-ui/app/cdap/components/HorizontalCarousel/index.tsx

```diff
@@ -0,0 +1,198 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import IconButton from '@material-ui/core/IconButton';
+import KeyboardArrowRight from '@material-ui/icons/KeyboardArrowRight';
+import KeyboardArrowLeft from '@material-ui/icons/KeyboardArrowLeft';
+import throttle from 'lodash/throttle';
+
+const styles = (): StyleRules => {
+  return {
+    contentContainer: {
+      paddingLeft: '60px',
+      paddingRight: '60px',
+      whiteSpace: 'nowrap',
+      overflowX: 'scroll',
+      scrollSnapType: 'x proximity',
+      scrollPadding: '60px',
+      display: 'flex',
+      scrollBehavior: 'smooth',
+      '-ms-overflow-style': 'none',
+      '&::-webkit-scrollbar': {
+        display: 'none',
+      },
+      '&:after': {
+        content: '""',
```

Comment:
Why do we need this?

Reply:
with flexbox, we need a content before height is applied